windows-gaming-driver/src/setup/vfio.rs

```diff
@@ -79,6 +82,75 @@ pub fn setup(setup: &mut SetupConfig, machine: &mut MachineConfig) -> bool {
     true
 }
 
+fn select(machine: &mut MachineConfig) -> Result<()> {
+    let udev = Context::new().expect("Failed to create udev context");
+    let mut iter = Enumerator::new(&udev)?;
+    iter.match_subsystem("pci")?;
+    let pci_devs: Vec<_> = iter.scan_devices()?.map(PciDevice::new).collect();
+
+
+    // filter to the display controller class (0x03XXXX, udev drops the leading zero)
+    select_device(machine, "Which graphics card would you like to pass through?", &pci_devs,
+                  |x| x.pci_class.starts_with("3") && x.pci_class.len() == 5)?;
+    if ask::yesno("Would you like to pass aditional PCI devices?") {
+        println!();
+        println!("Please choose aditional pci devices. These will be passed through to the VM on boot.");
```

Comment:
Well how should something be passed to the vm on (linux) boot when the vm is not running?

Although we could indeed word it better.